src/autopas/AutoPasDecl.h

```diff
@@ -458,6 +458,12 @@ class AutoPas {
    */
   [[nodiscard]] std::array<double, 3> getBoxMax() const;
 
+  /**
+   * get the bool value indicating if the search space is trivial (not more than one configuration to test).
+   * @return bool indicating if search space is trivial.
+   */
+  [[nodiscard]] bool getsearchSpaceIsTrivial();
```

Comment:
I think I would actually just call it `searchSpaceIsTrivial()`

Reply:
Completed!